src/hooks/usePairVolume24hUSD.ts

```diff
@@ -13,7 +13,7 @@ import { SWPRSupportedChains } from '../utils/chainSupportsSWPR'
 
 const GET_BLOCK_BY_TIMESTAMP = gql`
   query getBlockFromTimestamp($timestamp: String!) {
-    blocks(first: 10, orderBy: number, orderDirection: asc, where: { timestamp_gt: $timestamp }) {
+    blocks(first: 999, orderBy: number, orderDirection: asc, where: { timestamp_gt: $timestamp }) {
```

Comment:
This solution seems to fetch subgraph data but don't know why! @adamazad any idea? Also does it look good to you?